docs/integrations/airflow/extractors/custom-extractors.md

```diff
@@ -4,14 +4,11 @@ sidebar_position: 1
 
 # Custom extractors
 
-OpenLineage integration works by detecting which Airflow operators your dag is using, and extracting lineage 
-data from them using extractors. 
+OpenLineage integration works by detecting which Airflow operators your dag is using, and extracting lineage data from them using extractors. 
```

Comment:
"This integration works by detecting which Airflow operators your DAG is using, and extracting lineage data from them using corresponding extractors."

(I know this PR just reformats this section, feel free to reject this comment and I'll address in separate PR)